src/frontend/src/lib/api/idb.api.ts

```diff
@@ -1,13 +1,15 @@
 import { browser } from '$app/environment';
+import { IDB_ADDRESS_STORE, IDB_ETH_ADDRESS_STORE } from '$lib/constants/idb.constants';
 import type { IdbEthAddress } from '$lib/types/idb';
 import type { Principal } from '@dfinity/principal';
 import { isNullish } from '@dfinity/utils';
 import { createStore, del, get, set, update, type UseStore } from 'idb-keyval';
 
 // There is no IndexedDB in SSG. Since this initialization occurs at the module's root, SvelteKit would encounter an error during the dapp bundling process, specifically a "ReferenceError [Error]: indexedDB is not defined". Therefore, the object for bundling on NodeJS side.
-const oisyEthAddressesStore = browser
-	? createStore('oisy-eth-addresses', 'eth-addresses')
-	: ({} as unknown as UseStore);
+const oisyAddressesStore = (storeName: string) =>
+	browser ? createStore(IDB_ADDRESS_STORE, storeName) : ({} as unknown as UseStore);
```

Comment:
Are you sure this work? i.e. having a unique name here but different store name?

Reply:
I double-checked and you are right, I confused it with [IDB](https://www.npmjs.com/package/idb): in our case, [it won't work](https://github.com/jakearchibald/idb-keyval/blob/main/custom-stores.md#defining-a-custom-database--store-name).

I will adapt the code to have different DB names too